entrypoint.sh

```diff
@@ -117,6 +117,7 @@ ibmcloud_powervs() {
 libvirt() {
     test_vars LIBVIRT_URI
 
+    [[ "${DISABLECVM}" ]] && optionals+="-disable-cvm "
```

Comment:
`DISABLECVM` can be "", `true` or `false`. Here you should compare with "" and `false`.

Reply:
Shouldn't I compare equality against true? The double negatives are messing with me lol.

Reply:
oh, yes, compare with true...I got confused too :)